be/src/storage/storage_engine.h

```diff
@@ -378,6 +386,14 @@ class StorageEngine {
 
     HeartbeatFlags* _heartbeat_flags = nullptr;
 
+    std::unordered_map<int64_t, std::pair<int64_t, int64_t>*> _tableid_auto_increment_map;
+
+    std::unordered_map<int64_t, std::mutex*> _tabletid_auto_increment_mutex;
```

Comment:
put mutex and pair<int64,int64> into a new object and provide a thread-safe get(alloc)_interval API

Reply:
have fixed it